### ISSUE TITLE
Occ-linemod

### ISSUE THREAD

**Issue by hillaric** [ethnhe/PVN3D#74]
First, congratulation for your new work FFB6D. I see that you test the occ_linemod in both methods, and I want to test PVN3D on occ_linemod, but I don't have the preprocessed occ-linemod data. In detail, the downloaded dataset can not be used directly, how do you process the data?

**Comment by ethnhe** [comment]
We use preprocessing code from PVNet to transform the pose parameter ([pvnet_code](https://github.com/zju3dv/pvnet/blob/48e5066668f7563434373bc909b842a7bd94b7b9/lib/utils/data_utils.py#L539)) and then use similar scripts as in our datasets/linemod/linemod_dataset.py to generate input to our model.

**Comment by hillaric** [comment]
thank you! I will have a try.

**Comment by hillaric** [comment]
The 'blender K' in common.py is the same K used in the 'occ_linemod'? I am a little confused.

**Comment by ethnhe** [comment]
No, both the linemod and occ_linemod use 'linemod' K, the 'blender' K is only used for the synthetic dataset.

**Comment by hillaric** [comment]
Would you mind provide the corresponding code, especially the gt.yml.  I try, but I can't do it.
![image](https://user-images.githubusercontent.com/50513602/112002067-7d462a80-8b5a-11eb-866c-3e67a47d87fc.png)

**Comment by hillaric** [comment]
After using the code 'Pose_transformer' you released in 'base_utils.py' for OCCLUSION_LINEMOD, I got a different pose from PVNet. Is it right?

**Comment by ethnhe** [comment]
You can check if the pose is correct by transforming the object vertexes or the selected keypoint by the pose parameters and project them back to the RGB image for visualization, as what we have done in ```datasets/linemod/linemod_dataset.py``` [code link](https://github.com/ethnhe/PVN3D/blob/b8cd1d8996ee25f1bc357c023ff9fb202e07c438/pvn3d/datasets/linemod/linemod_dataset.py#L394).

**Comment by aiai84** [comment]
hello, have you solve the problem, I want to test the OCC-Linemod dataset, could you show your change or addition on the original code? Thank you

**Comment by William-Klay** [comment]
For occ-lm, i found the model performance on eggbox is pretty bad. Can you please give me any insight about it?

**Comment by d-serafly** [comment]
> For occ-lm, i found the model performance on eggbox is pretty bad. Can you please give me any insight about it?

May I know how you completed the test on occ? Especially in occ dataset processing.

**Comment by weidu3** [comment]
> For occ-lm, i found the model performance on eggbox is pretty bad. Can you please give me any insight about it?

Yes, I also found this problem, eggbox in the  LM-O dataset has been suppressed by a red car, and I can't reach the add-s in the article with the pre-training weights provided by the author or training myself. So I'm curious, eggbox is testing on the LM-O dataset, is it using the original data of LM-O?
